.ci/dockerImagesOpbeans.groovy

```diff
@@ -182,10 +182,18 @@ def buildDockerImage(args){
       withEnv(env){
         retry(3) {
           sleep randomNumber(min: 5, max: 10)
-          sh(label: "build docker image", script: "docker build ${options} -t ${image} .")
+          if(isUnix()){
+            sh(label: "build docker image", script: "docker build ${options} -t ${image} .")
+          } else {
+            bat(label: "build docker image", script: "docker build ${options} -t ${image} .")
+          }
```

Comment:
You can use https://github.com/elastic/apm-pipeline-library/blob/master/vars/cmd.groovy

```suggestion
          cmd(label: "build docker image", script: "docker build ${options} -t ${image} .")
```